client/src/app/_services/login.service.ts

```diff
@@ -110,10 +115,11 @@ export class LoginService {
                     });
                 }
             })
-            .catch(this.handleError);
+            .catch(this.handleError.bind(this));
     }
 
     private handleError(err: HttpErrorResponse) {
+        console.log(err);
```

Comment:
`console.error` may be better

Reply:
Yes. By the way, it will change when #126 will be fixed.